.azure-pipelines/templates/jobs/extended-tests-jobs.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           PYTHON_VERSION: 3.11
           TOXENV: py311
         linux-isolated:
-          TOXENV: 'isolated-{acme,certbot,apache,cloudflare,digitalocean,dnsimple,dnsmadeeasy,gehirn,google,linode,luadns,nsone,ovh,rfc2136,route53,sakuracloud,nginx}'
+          TOXENV: 'isolated-acme,isolated-certbot,isolated-apache,isolated-cloudflare,isolated-digitalocean,isolated-dnsimple,isolated-dnsmadeeasy,isolated-gehirn,isolated-google,isolated-linode,isolated-luadns,isolated-nsone,isolated-ovh,isolated-rfc2136,isolated-route53,isolated-sakuracloud,isolated-nginx'
```

Comment:
what was the undocumented thing this refers to??

Reply:
sometimes when you rewrite your entire project from scratch you lose the ability to expand environment names passed in from the cli 🤷🤷🤷